morpheus/pipeline/output/to_file.py

```diff
@@ -120,7 +120,7 @@ def _build_single(self, seg: neo.Segment, input_stream: StreamPair) -> StreamPai
             def node_fn(input: neo.Observable, output: neo.Subscriber):
 
                 # Ensure our directory exists
-                os.makedirs(os.path.dirname(self._output_file), exist_ok=True)
+                os.makedirs(os.path.realpath(os.path.dirname(self._output_file)), exist_ok=True)
```

Comment:
Isnt this backwards. Shouldnt it be 
```
os.makedirs(os.path.dirname(os.path.realpath(self._output_file)), exist_ok=True)
```

Reply:
This leads me to believe that your test might not be working as intended

Reply:
As far as I can tell the output is always the same, if you prefer one over the other let me know.

```
In [1]: import os

In [2]: os.path.realpath(os.path.dirname('README.md'))
Out[2]: '/home/dagardner/work/morpheus'

In [3]: os.path.realpath(os.path.dirname('README.md')) == os.path.dirname(os.path.realpath('README.md'))
Out[3]: True

In [4]: os.path.realpath(os.path.dirname('./README.md')) == os.path.dirname(os.path.realpath('./README.md'))
Out[4]: True

In [5]: os.path.realpath(os.path.dirname('./examples/log_parsing/README.md')) == os.path.dirname(os.path.realpath('./examples/log_parsin
   ...: g/README.md'))
Out[5]: True
```